examples/scalajs-react/src/main/scala/todomvc/CTodoList.scala

```diff
@@ -70,62 +83,64 @@ object CTodoList {
           <.input(
             ^.className     := "new-todo",
             ^.placeholder   := "What needs to be done?",
-            ^.onKeyDown  ~~>? handleNewTodoKeyDown _,
+            ^.onKeyDown   ==>? cbs.value().handleNewTodoKeyDown,
```

Comment:
I see! I didn't realize that this had more effects than just retrieving the value from the state. In that case, this is totally justified.